### DISCUSSION TITLE
Get the coordinates of center of mass in 3D scan

### DISCUSSION THREAD

**Discussion by andre-fs-ferreira** [Project-MONAI/tutorials#1080] [Q&A] [answered]
Hello everyone!

I would like to know how to get the center of mass coordinates of a label (e.g. tumor label center of mass), then specify the center and ROI and cut the 3D scan (e.g. brain scan) into as same coordinates.

Thank you very much!

**Comment by KumoLiu** [comment, marked as answer]
Hi @ShadowTwin41, you could do something like this:
```
CropForegroundd(keys=["image", "label"], source_key="label")
```
Hope it can help you, thanks!

**Reply by andre-fs-ferreira** [reply]
Thank you very much! It works, but I also want a little bit of surrounding tissue, not just the tumor. Is it possible?

**Reply by wyli** [reply]
yes, `CropForegroundd` can take a `margin` parameter for that

**Reply by andre-fs-ferreira** [reply]
Excellent! Thank you very much. Just one more question, is it possible to have a specific size, or it is better to use `CenterSpatialCropd ` afterwards?

**Reply by KumoLiu** [reply]
Yes, I think you could use a `ResizeWithPadOrCrop` afterward to ensure the same shape.
